Tests/Client/Integration Tests/ChannelTests01+Upload.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ChannelTests01+Upload.swift
+//  StreamChatClientTests
+//
+//  Created by Alexey Bukhtin on 17/01/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+@testable import StreamChatClient
+
+final class ChannelTests01_Upload: TestCase {
+    
+//    func test01ImageUpload() {
```

Comment:
Why is it commented?

Reply:
It needs a little bit of another implementation. I checked with some local data.

Reply:
It's in TODO